src/assets/utils/birthDayValidation.js

```diff
@@ -0,0 +1,13 @@
+const isAdult = (inputDate) => {
+  const today = new Date();
+  const eighteenYearsAgo = new Date(
+    today.getUTCFullYear() - 18,
+    today.getMonth(),
+    today.getUTCDate()
+  );
```

Comment:
That might be not exactly doing what you need as it will still process with current timezone.

Maybe using utc to parse would be better:
```suggestion
  const eighteenYearsAgo = new Date(Date.UTC(
    today.getFullYear() - 18,
    today.getMonth(),
    today.getDate()
  ));
```

Reply:
corrected